docs/resources/aws_iam_managed_policies.md

```diff
@@ -0,0 +1,76 @@
+---
+title: About the aws_iam_managed_policies Resource
+platform: aws
+---
+
+# aws\_iam\_managed\_policies
+
+Use the `aws_iam_managed_policies` InSpec audit resource to test properties of a collection of AWS IAM Managed Policies.
+
+## Syntax
+
+The `aws_iam_managed_policies` resource returns a collection of IAM managed policies and allows testing of that collection.
+
+    describe aws_iam_managed_policies do
+      its('policy_names') { should include('POLICY_NAME') }
+    end
+
+This resource allows filtering by scope.
+To list only AWS managed policies, set `Scope` to `AWS`. To list only the customer managed policies in your AWS account, set `Scope` to `Local`. If scope is not supplied `ALL` policies are returned.
```

Comment:
```suggestion
To list only AWS-managed policies, set `Scope` to `AWS`. To list only the customer-managed policies in your AWS account, set `Scope` to `Local`. If a scope is not supplied, `ALL` policies are returned.
```

Reply:
Can you add an example of this.

I assume it's something like: `describe aws_iam_managed_policies(scope: 'aws' ) do`

Reply:
Thanks, Adding